web/src/features/notifications/NotificationWrapper.tsx

```diff
@@ -103,11 +103,24 @@ const exitAnimationBottom = keyframes({
   },
 });
 
+const durationCircle = keyframes({
+  '0%': { strokeDasharray: `0, ${15.1 * 2 * Math.PI}` },
+  '100%': { strokeDasharray: `${15.1 * 2 * Math.PI}, 0` },
+});
+
+const isValidHexColor = (hex: string) => {
+  return /^#[0-9A-F]{6}$/i.test(hex)
+}
```

Comment:
Hex or mantine color aren't the only options, you can pass colors as rgb or hsl like such: `rgb(255, 255, 110)` which results in the icon not being colored

![image](https://github.com/overextended/ox_lib/assets/39926192/40dc8bfb-9358-4ce1-b9df-413b38d1a647)

Reply:
Oh yes, i clearly forgot about this